cassovary-core/src/main/scala/com/twitter/cassovary/graph/SortedNeighborsNodeOps.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2014 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.graph
+
+import com.twitter.cassovary.util.SortedArrayOps
+
+trait SortedNeighborsNodeOps {
+  self: Node =>
+
+  override protected def containsNode(nodeIds: Seq[Int], queryNodeId: Int): Boolean = {
```

Comment:
Interesting. it works in my repl:

scala> trait A { protected def f(s: Seq[Int]): Boolean = true }
defined trait A

scala> class A2 extends A { protected def f(a: Array[Int]) = false 
     | println(f(Array(1)))
     | }
defined class A2

scala> val a2 = new A2
false
a2: A2 = A2@4f2c9ba6

scala> object A3 extends A2 { println("In Seq:" + f(Seq(1,2))); println("In Array:" + f(Array(0))) }
defined module A3

scala> A3
false
In Seq:true
In Array:false
res5: A3.type = A3$@4a9f80d3